### PR TITLE
Add store overrides and debug flag in `create_forge`

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

**Review by mgeplf — APPROVED** [review]

File: bluepyentity/environments.py [hunk 1]
```diff
@@ -18,13 +20,81 @@ def get_environment(env):
...
+        allowed_keys = {
+            "endpoint",
+            "searchendpoints",
+            "params",
+            "versiond_id_template",
+            "file_resource_mapping",
+        }
+        if not all(key in allowed_keys for key in store_overrides):
+            raise BluepyEntityError(
+                f"Unsupported 'store_overrisdes' keys. "
```
Comment (joni-herttuainen) [inline bluepyentity/environments.py]: typo -> store_overrides

File: tests/test_environments.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import pytest
+from bluepyentity.exceptions import BluepyEntityError
+from bluepyentity import environments as tested
+
+
+def test_create_forge__raises():
+
+    with pytest.raises(BluepyEntityError):
```
Comment (joni-herttuainen) [inline tests/test_environments.py:8]: Maybe, for future-proofing, we should add match="Unsupported 'store_overrides' keys"

**Review by joni-herttuainen — APPROVED** [review]
some nit-picking but good-to-go

File: bluepyentity/environments.py (hunk 1 above)
Comment (eleftherioszisis) [reply bluepyentity/environments.py]: Fixed.

File: tests/test_environments.py (hunk 2 above)
Comment (eleftherioszisis) [reply tests/test_environments.py]: Added.

**Pull request merged**